### PR TITLE
Added Brazilian Portuguese localization

### PR DESCRIPTION
**Pull request by EvertonCa**
Added Brazilian Portuguese localization

### CODE REVIEW

File: IntentsExtension/pt-BR.lproj/Intents.strings [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"89XfI6" = "Is Dark Mode";
```
Comment (jacklandrin) [inline IntentsExtension/pt-BR.lproj/Intents.strings:1]: this file isn't needed. 🙂

File: README.md [hunk 2]
```diff
@@ -227,16 +227,16 @@ S-GPT is already in Shortcuts gallery. Easily ask any questions to chatGPT via S
...
+| Language | Contributor              | Language        | Contributor    |
+|----------|--------------------------|:----------------|:---------------|
+| German   | @C0d3Br3aker             | Italian         | @bellaposa     |
+| Croatian | @milotype                | Russian         | @kirillyakopov |
+| Turkish  | @berkbatuhans            | Spanish         | @kant          |
+| Polish   | @kpacholak               | Japanese        | @ShogoKoyama   |
+| Dutch    | Alex                     | Somali          | @abdorizak     |
+| Filipino | Rosel                    | Korean          | @iosdevted     |
+| French   | @BtKent and Ange Lefrère | Ukrainian       | @andryua       |
+| Slovak   | @Svec-Tomas              | Portuguese (BR) | @EvertonCa     |
```
Comment (jacklandrin) [inline README.md:239]: Please also add Portugusee (BR) in `Supported Languages 🇺🇳` Section, scroll up on README, you will find it.

**Review by jacklandrin — COMMENTED** [review]
@EvertonCa Thank you for your contribution! I left few comments, please check them. 🙂

**Comment by EvertonCa** [comment]
> @EvertonCa Thank you for your contribution! I left few comments, please check them. 🙂

All done!

File: OnlySwitch.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -689,6 +689,8 @@
 		1081EF19287F28C5005849B5 /* so */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = so; path = so.lproj/Intents.strings; sourceTree = "<group>"; };
 		1081EF1A287F28C5005849B5 /* so */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = so; path = so.lproj/Localizable.strings; sourceTree = "<group>"; };
 		29107C0828111D3A00E88A86 /* ja */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ja; path = ja.lproj/Localizable.strings; sourceTree = "<group>"; };
+		40109DE32B6B18E0005D4802 /* pt-BR */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = "pt-BR"; path = "pt-BR.lproj/Intents.strings"; sourceTree = "<group>"; };
```
Comment (jacklandrin) [inline OnlySwitch.xcodeproj/project.pbxproj:692]: When you remove the Intents localization file, this line should be also removed.

**Review by jacklandrin — APPROVED** [review]

**Pull request merged**